src/java/org/apache/cassandra/metrics/ClientMetrics.java

```diff
@@ -20,12 +20,20 @@
 
 import java.util.*;
```

Comment:
could you expand these imports?

Reply:
i'll do that in ClientsTable as well as noticed a wildcard import there as well.